src/status_im/transport/core.cljs

```diff
@@ -34,10 +35,19 @@
 (fx/defn fetch-node-info-fx [cofx]
   {::fetch-node-info []})
 
+(fx/defn init-messenger [cofx]
```

Comment:
would be worth adding here as a NOTE why this is all necessary

Reply:
done